CHANGES.md

```diff
@@ -1,3 +1,8 @@
+2017-07-21, Version 3.3.0
+=========================
+
+ * Add Node.js 8.2.1 support (Oleg Kubrakov)
```

Comment:
This file is autogenerated as part of our release process, please revert these changes.